datanode/gateway/graphql/schema.graphql

```diff
@@ -5462,6 +5490,18 @@ type LiquidityProvisionsConnection {
   pageInfo: PageInfo!
 }
 
+"Edge type containing the liquidity provision and cursor information returned by a LiquidityProvisionsWithPendingConnection"
```

Comment:
I renamed the edge type because an edge has a singular Liquidity Provision, while the Connection I left as Provisions (plural) because the connection can return a list.